docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: '3'
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: "Dockerfile.api"
+    command: python3 ohp.py --start-api-server
+    environment:
+      - FLASK_ENV=development
+      - DOCKER_ENV=1
```

Comment:
good to have an env to clearout we are using docker, but maybe something more clear `ENV=docker` or `DOCKER_ENV=true`

Reply:
also we need to make sure such env var will not be use in images that user can read them, like weak ssh password

Reply:
Good point! I'll look into it.

Reply:
I checked and this env variable is not getting passed on to the docker images of the modules. I have updated the environment variable.

Reply:
@ChakshuGupta as an example, you need to add it to docker options.

https://github.com/zdresearch/OWASP-Honeypot/blob/master/modules/ssh/strong_password/__init__.py#L50-L63
current code:
```
def module_configuration():
    """
    module configuration
    Returns:
        JSON/Dict module configuration
    """
    return {
        "username": "root",
        "password": generate_token(16),
        "extra_docker_options":
        ["--volume {0}/tmp:/root/logs/".format(os.getcwd())],
        "module_processor": ModuleProcessor()
    }
```
possible fix suggestion:
```python
def module_configuration():
    """
    module configuration
    Returns:
        JSON/Dict module configuration
    """
    return {
        "username": "root",
        "password": generate_token(16),
        "extra_docker_options":
        ["--volume {0}/tmp:/root/logs/".format(os.getcwd()), "-e DOCKER_ENV" if whatever else ""],
        "module_processor": ModuleProcessor()
    }
```

is that what you meant/needed?

Reply:
@Ali-Razmjoo :  this PR looks good to me. only thing is credential events are not getting logged in. Could you help here?
Network event's were also not getting logged in by default then when we changed the ignore_real_machine_ip_address to True it's logs.